### ISSUE TITLE
W3 A11y Standard: Screen Reader Support

### ISSUE THREAD

**Issue by tamascsaba** [swimlane/angular2-data-table#257]
The grid is entirely dom based and as such does not support a11y.
In order for screen readers to correctly read out table content the dom must have the correct role tags on the grid/rows/cells.
Have a look at the W3 standards: http://www.w3.org/TR/wai-aria/

**Comment by amcdnl** [comment]
Agreed, I don't have a big driver for this right now but happy to accept PRs.

**Comment by deebloo** [comment]
@amcdnl I can probably start talking a look at this

**Comment by amcdnl** [comment]
I have some basic keyboard support for selection of cells/rows.

**Comment by ParadoxZero** [comment]
Is there any work happening on this?

**Comment by amyspeed** [comment]
Where are we on this? It has been over 4 years and this does not look to be resolved.

**Comment by Kodak1234** [comment]
What's the update on the accessibility issue?
